bms/bms_app/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf.urls import url
+
+from django.conf.urls import url, include
+from django.contrib import admin
+from .views import home, register
+from .views import product_home_page
+from django.contrib.auth import views as auth_views
+from django.contrib.auth.views import login, logout
+from . import views
+
+urlpatterns = [
+    url(r'^$', home),
+    url(r'^register/', register),
+    url(r'^login/', login),
+    url(r'^home_page/',views.product_home_page),
```

Comment:
1st home url is my landing home page
last url is my home page, when user successfully logged in then it will redirect to home page